docs/_layouts/documentation.html

```diff
@@ -23,5 +23,10 @@
 </div>
 
 <script src="https://cdn.jsdelivr.net/instantsearch.js/0/instantsearch.js"></script>
+{% if site.env == 'development' %}
+  <!-- override with the latest/dev version if possible -->
+  <!-- that's required while creating doc of new (non-published) widgets -->
+  <script src="http://localhost:8080/instantsearch.js"></script>
```

Comment:
> Should we use /instantsearch.js (so that ngrok etc will continue to work)

Nope, not the same port.

> Should we do a true if else to avoid having two instances here (2* react = issue)

It's working but we could do:
 - load from :8080